addons/account/models/account_tax.py

```diff
@@ -282,8 +282,17 @@ def _check_children_scope(self):
         for tax in self:
             if not tax._check_m2m_recursion('children_tax_ids'):
                 raise ValidationError(_("Recursion found for tax '%s'.") % (tax.name,))
-            if any(child.type_tax_use not in ('none', tax.type_tax_use) or child.tax_scope != tax.tax_scope for child in tax.children_tax_ids):
-                raise ValidationError(_('The application scope of taxes in a group must be either the same as the group or left empty.'))
+            for child in tax.children_tax_ids:
+                if child.type_tax_use not in ('none', tax.type_tax_use):
+                    raise ValidationError(_(
+                        "Tax %s has not the same type_tax_use of tax %s: %s => %s",
+                        tax.name, child.name, tax.type_tax_use, child.type_tax_use
+                    ))
+                if tax.tax_scope not in (False, child.tax_scope):
```

Comment:
Ask the team what to do when one of the child taxes has a `tax_scope` and the rest hasn't.